examples/AWS_MQTT_Config/AWS_MQTT_Config.ino

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2017, Helium Systems, Inc.
+ * All Rights Reserved. See LICENCE.txt for license information
+ */
+
+#include "Arduino.h"
+#include "Board.h"
+#include "Helium.h"
+#include "HeliumUtil.h"
+#include "ArduinoJson.h"
+
+// NOTE: This sample uses the "AWS IoT" channel which is available
+// on the Helium Dashboard. To use another channel, change the channel
+// name to the target channel and adjust the Config variable name to match
+// the cloud service's variable name
+//
+// ***AWS IoT*** Once the application is running on the device, change the global
+// config variable "interval" in the Device Shadow.
+//
+// After one previous sleep cycle you should see the message arrive at
+// the new interval rate (in milliseconds)
+//
+#define CHANNEL_NAME "YOUR CHANNEL NAME HERE"
+#define CONFIG_INTERVAL_KEY "channel.interval_ms"
+
+Helium  helium(&atom_serial);
+Channel channel(&helium);
+Config  config(&channel);
+
+int32_t send_interval;
+String message = "Hello Helium!";
+
+void
+update_config(bool stale)
+{
+    if (stale)
+    {
+        DBG_PRINT(F("Fetching Config - "));
+        int status = config.get(CONFIG_INTERVAL_KEY, &send_interval, 5000);
+        report_status(status);
+
+        if (status == helium_status_OK)
+        {
+            DBG_PRINT(F("Updating Config - "));
+            status = config.set(CONFIG_INTERVAL_KEY, send_interval);
+            report_status(status);
+        }
+    }
+}
+
+void
+setup()
+{
+    Serial.begin(9600);
+    DBG_PRINTLN(F("Starting"));
+
+    // Begin communication with the Helium Atom The baud rate differs
+    // per supported board and is configured in Board.h
+    helium.begin(HELIUM_BAUD_RATE);
+
+    // Connect the Atom to the Helium Network
+    helium_connect(&helium);
+    // and do a channel connect
+    channel_create(&channel, CHANNEL_NAME);
+
+    // Get the initial interval
+    update_config(true);
+}
+
+void
+loop()
+{
+    StaticJsonBuffer<JSON_OBJECT_SIZE(2) + 100> jsonBuffer;
```

Comment:
Go into the arduinojson size tool and see how many bytes you actually need.. 100 is way to many for just one object with one integer key/value pair